song-server/src/main/java/bio/overture/song/server/model/analysis/Analysis.java

```diff
@@ -69,6 +61,17 @@
   @Column(name = TableAttributeNames.STATE, nullable = false)
   private String analysisState = UNPUBLISHED.name();
 
+  @Column(name = TableAttributeNames.CREATED_AT, nullable = false)
+  @CreationTimestamp
+  private LocalDateTime createdAt;
+
+  @Column(name = TableAttributeNames.UPDATED_AT, nullable = false)
+  @UpdateTimestamp
+  private LocalDateTime updatedAt;
+
+  @Transient private LocalDateTime firstPublishedAt;
+  @Transient private LocalDateTime publishedAt;
```

Comment:
make sure to add these fields to bio.overture.song.core.model.Analysis (the DTO). This file is used by client, so it can deserialize properly.

Reply:
I did this, but... This DTO you have defined is not used anywhere, I'm pretty sure. I think adding it to the Controller tests is the first usage of it.